src/sas/qtgui/Perspectives/Corfunc/CorfuncPerspective.py

```diff
@@ -77,14 +84,17 @@ def draw_real_space(self):
         self.axes = self.fig.add_subplot(111)
         self.axes.set_xscale("linear")
         self.axes.set_yscale("linear")
+        self.axes.set_xlabel("Z [$\AA$]")
+        self.axes.set_ylabel("Correlation")
+        self.fig.tight_layout()
```

Comment:
Would also be great if the plots had titles as it makes it difficult to tell which graph is showing what, especially when no data is loaded.